README.md

```diff
@@ -86,315 +54,86 @@ By focusing on the re-use of standard protocols, formats and vocabularies, and
+| `GET` | `/path/to/resource` | Retrieve a resource or list of resources in a container |
+| `POST` | `/path/to/parent/` | Create a new resource in container |
+| `PUT` | `/path/to/resource` | Replace a resource |
+| `PATCH` | `/path/to/resource` | Update a resource |
+| `DELETE` | `/path/to/resource` | Delete a resource |
 
-- the original XML/RDF format
-- Turtle RDF (easier to read and write, used by default in Solid)
-- RDFa (RDF embedded into HTML attributes)
-- JSON-LD
+The data that is sent and retrieved is RDF; the RDF serialisation we use by default is
+Turtle. Don't worry if you're not familiar with this format, we're going to show you
+handy libraries for working with it. If you want to find out more about Linked Data
+principles you can see **[Linked Data 101](http://rhiaro.github.io/sws/)** for a quick
+run down.
 
-See **[Linked Data 101](http://rhiaro.github.io/sws/)** or the [RDF primer](https://www.w3.org/2000/10/swap/Primer.html) for a quick introduction
-to RDF structure and concepts.
+This API is a W3C standard called [Linked Data Platform
+(LDP)](http://www.w3.org/TR/ldp/) (see also [LDP Primer](http://www.w3.org/TR/ldp-primer/)).
 
 ### Schemas and Vocabularies
 
-As much as possible, Solid apps should aim to use existing schemas and vocabularies
-to promote [interoperability](#interoperability).
+You're used to describing data in a database with a schema; a set of terms that describe
+attributes of a data item for which you can set values. For global interoperability on the
+Web, we all need to find a way to cooperate when choosing schemas to represent our data.
+
+Fortunately many people publish RDF vocabularies which cover different domains, and the idea
+is to reuse existing ones as much as possible. All terms in a vocabulary (just like any other data
+item) have their own URL. This ensures that when two people use the same term they can use a
+globally unique identifier to show they mean the same thing.
 
-**Vocabularies** or ontologies are a set of words and phrases that someone has decided to put together for the
-purposes of describing a particular topic. They usually consist of classes and properties.
+* **Vocabularies** or ontologies are a set of words and phrases that someone has decided to put
+together for the purposes of describing a particular topic. They usually consist of classes and
+properties.
 
-**Classes** are used to say that `<some-thing>` is of type `<other-thing>`.
-For example:
-`<http://rhiaro.co.uk/about#me> type <Person>`.
-(Person is the class). Classes start with an uppercase letter.
+* **Classes** are used to say that `<some-thing>` is of type `<other-thing>`.
+For example: `<http://rhiaro.co.uk/about#me> type <Person>`. (Person is the class).
 
-**Properties** are just the describing words. In the previous example `type` is
-the property, and before that name was the property. Properties are camelCased.
+* **Properties** are just the describing words. In the previous example `type` is
```

Comment:
> **Properties** are just the describing words

Not sure about this phrasing, either. Properties are more than describing words -- if anything, maybe you can connect them to the *attributes* you mention earlier.